### PR TITLE
Migrate services from superduperdb to private

### PR DESCRIPTION
**Pull request by kartik4949**
## Checklist

- [ ] Is this code covered by new or existing unit tests or integration tests?
- [ ] Did you run `make unit_testing` and `make integration-testing` successfully?
- [ ] Do new classes, functions, methods and parameters all have docstrings?
- [ ] Were existing docstrings updated, if necessary?
- [ ] Was external documentation updated, if necessary?

### CODE REVIEW

**Review by blythed — CHANGES_REQUESTED** [review]
The workflows and `Makefile` will break with this. Please migrate also.

File: .github/workflows/ci_code.yml [hunk 1]
```diff
@@ -113,12 +109,6 @@ jobs:
           # Run the test-suite within the sandbox image from the test environment
           docker exec testenv-sandbox-1 sh -c 'make databackend_testing'
```
Comment (blythed) [inline .github/workflows/ci_code.yml:110]: This is now only used for mongodb testing, the whole sandbox build, etc., is overkill. We should simplify.

Comment (kartik4949) [reply .github/workflows/ci_code.yml:110]: can we do it as separate pr? 
else the pr will be overloaded with big changes

**Review by blythed — APPROVED** [review]
Thanks!

**Pull request merged**